docs/guides/create_update_solution.md

```diff
@@ -0,0 +1,339 @@
+   powerbi:
+    workspace:
+     name: "My workspace Powerbi Name"
+     reports:
+      - name: Report Name A
+        type: dashboard
+        path: "powerbi/myreportA.pbix"
+     permissions:
+     - identifier: "e-mail@cosmotech.com"
+       rights: Admin
+       type: User
+   adx:
+    database:
+     create: true
+     retention: 365
+     permissions:
+     - type: User
+       email: e-mail@cosmotech.com
+       principal_id: "412f3fad-3ce3-410a-994c-2a36bccaa0b2"
+       role: Admin
+     scripts:
+     - id: "demoscript"
+       name: Create.kql
+       path: "adx/scripts"
+   eventhub:
+    consumers:
+    - displayName: adx
+      entity: ProbesMeasures
+    - displayName: adx
+      entity: ScenarioMetadata
+    - displayName: adx
+      entity: ScenarioRun
+    - displayName: adx
+      entity: ScenarioRunMetadata
+    connectors:
+     - table_name: ProbesMeasures
+       consumer_group: adx
+       connection_name: ProbesMeasures
+       database_target: "{{services['api.organization_id']}}-{{key}}"
+       format: JSON
+       compression: Gzip
+       mapping: ProbesMeasuresMapping
+     - table_name: ScenarioMetadata
+       consumer_group: adx
+       connection_name: ScenarioMetadata
+       database_target: "{{services['api.organization_id']}}-{{key}}"
+       format: CSV
+       compression: None
+       mapping: ScenarioMetadataMapping
+     - table_name: SimulationTotalFacts
+       consumer_group: adx
+       connection_name: ScenarioRun
+       database_target: "{{services['api.organization_id']}}-{{key}}"
+       format: JSON
+       compression: None
+       mapping: SimulationTotalFactsMapping
+     - table_name: ScenarioRunMetadata
+       consumer_group: adx
+       connection_name: ScenarioRunMetadata
+       database_target: "{{services['api.organization_id']}}-{{key}}"
+       format: CSV
+       compression: None
+       mapping: ScenarioRunMetadataMapping
+ payload:
+  key: "{{key}}"
+  name: "My Workspace Name"
+  description: "Workspace for solution"
+  solution:
+   solutionId: "{{services['api.solution_id']}}"
+  useDedicatedEventHubNamespace: true
+  sendScenarioMetadataToEventHub: true
+  sendInputToDataWarehouse: true
+  sendScenarioRunToEventHub: true
+  security:
+   default: none
+   accessControlList:
+   - id: e-mail@cosmotech.com
+     role: admin
+```
+Path to existing powerBI reports could be declared in `powerBI` section of sidecars; adx script must be
+in adx folder of your project.
+
+!!! abstract "Remember"
+
+    Some operations could fail if Babylon doesn't have enough rights to create Azure resources. 
+    To create automatically an adx database, an azure function or an event hub, Babylon must be at least on Contributor 
+    level. Owner rights allow it to assign roles to resources. If you security policy doesn't grant
+    such access to Babylon, these operations must be done manually. 
+
+To deploy a webapp you can use an existing app registration or use an existing one; it can be 
```

Comment:
Ici, j'ai supprimé "use an existing one", je crois, et dupliqué ?
To deploy a webapp you can use an existing app registration or creating new one; this can be declared by the `create` key of `sidecars.azure.app` section. If it is set to false, both a client_id and a name
for your app registration must be declared, if set to true, Babylon will create a new one.